src/js/modules/infragistics.ui.editors.js

```diff
@@ -8974,6 +8969,23 @@
 			}
 			return result;
 		},
+		_validateMinMax: function(date) {
```

Comment:
rename the method to _getValueBetweenMinMax, or similar, because it's not returning Boolean value -what is expected from the name.